### PR TITLE
Bug 1543408 Add search region pref to search_clients_daily

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1543408

Depends on https://github.com/mozilla/telemetry-batch-view/pull/535#

### CODE REVIEW

**Comment by jklukas** [comment]
My plan is to watch main summary roll out tomorrow and confirm the new column this depends on is available, then pursue merging this.

File: tests/test_search_aggregates.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ def generate_addon(addon_id, name, version):
     ("channel", "release", StringType(), True),
     ("country", "DE", StringType(), True),
     ("locale", "de", StringType(), True),
+    ("user_pref_browser_search_region", "DE", StringType(), False),
```
Comment (wlach) [inline tests/test_search_aggregates.py]: Should probably test this being null. You should be able to do that below around line 347

**Comment by wlach** [comment]
Oops, the unit tests are failing. You might need to make some minor modifications to the expected data structure. Should be straightforward.

**Comment by jklukas** [comment]
Yeah, I'm having trouble getting the tests to run successfully locally. Will iterate.

**Pull request merged**